### PR TITLE
Uml 1301 content change

### PR DESCRIPTION
**Pull request by SeemaMenon**
# Purpose

https://opgtransform.atlassian.net/browse/UML-1301

Fixes UML-1301

## Approach

_Explain how your code addresses the purpose of the change_

## Learning

_Any tips and tricks, blog posts or tools which helped you. Plus anything notable you've discovered about the Use a Lasting Power of Attorney service_

## Checklist

* [X] I have performed a self-review of my own code
* [ ] I have added relevant logging with appropriate levels to my code
* [ ] I have updated documentation (Confluence/GitHub wiki/tech debt doc) where relevant
* [ ] I have added tests to prove my work
* [ ] I have added welsh translation tags and updated translation files
* [ ] I have run an accessibility tool on any pages I have made changes to and fixed any issues found
* [ ] The product team have tested these changes

### CODE REVIEW

**Comment by SeemaMenon** [comment]
@GemTay  

As per your previous suggestion, I checked with Sarah and she has agreed on the content changes for the below:
Screenshot 2021-04-26 at 16 25 26

File: service-front/app/languages/cy/LC_MESSAGES/messages.po [hunk 1]
```diff
@@ -5546,7 +5526,7 @@ msgstr "Nodwch eich cod post"
 
 #: src/Actor/templates/actor/send-activation-key-confirmation.html.twig:18
 msgid "We're posting you an activation key"
-msgstr "Rydym yn anfon allwedd cadarnhau cyfrif atoch drwy’r post"
+msgstr "Rydym yn anfon allwedd cadarnhau cyfrif atoch"
```
Comment (GemTay) [inline service-front/app/languages/cy/LC_MESSAGES/messages.po:5529]: How come this welsh translation has changed?

File: service-front/app/src/Actor/templates/actor/request-activation-key.html.twig [hunk 2]
```diff
@@ -20,19 +20,15 @@
...
 
                     {{ govuk_form_open(form) }}
 
                         {{ govuk_form_element(form.get('__csrf')) }}
 
                         {{ govuk_form_element(form.get('opg_reference_number'), {
-                            'label': 'OPG reference number'| trans,
-                            'hint': 'OPG reference numbers are 12 numbers long.<br/>For example, 700000000000'| trans,
+                            'label': 'LPA reference number'| trans,
+                            'hint': 'LPA reference numbers are 12 numbers long <br/>For example, 7000-0000-0000'| trans,
```
Comment (GemTay) [inline service-front/app/src/Actor/templates/actor/request-activation-key.html.twig:31]: The label on the 'Check your answers' page is still 'OPG reference number', it it also needs changing to keep it consistent across the service

**Review by GemTay — COMMENTED** [review]
just a few tweaks, everything else looks good!

File: service-front/app/src/Actor/templates/actor/request-activation-key.html.twig (hunk 2 above)
Comment (SeemaMenon) [reply service-front/app/src/Actor/templates/actor/request-activation-key.html.twig:31]: Yes, That change is in my current ticket 1371. Else I could do those changes in here...Let me do that then!

File: service-front/app/languages/cy/LC_MESSAGES/messages.po (hunk 1 above)
Comment (SeemaMenon) [reply service-front/app/languages/cy/LC_MESSAGES/messages.po:5529]: That's right...didn't notice this. Changed now.

File: service-front/app/src/Actor/templates/actor/request-activation-key.html.twig (hunk 2 above)
Comment (SeemaMenon) [reply service-front/app/src/Actor/templates/actor/request-activation-key.html.twig:31]: There's the error messages too in relation to this fields to be corrected as well. Adding in here now.

**Comment by SeemaMenon** [comment]
The changes in here now clubs requirements in UML-1371 too.

https://opgtransform.atlassian.net/browse/UML-1371

**Comment by GemTay** [comment]
> @GemTay : The changes in here now clubs requirements in [UML-1371](https://opgtransform.atlassian.net/browse/UML-1371) too.
> 
> https://opgtransform.atlassian.net/browse/UML-1371

Probably best to do all of the changes with "OPG reference number" in one ticket, otherwise when either of these PR's gets merged, the labels, hint text or error messages will differ across the older LPA journey

File: service-front/app/src/Actor/templates/actor/lpa-add.html.twig [hunk 3]
```diff
@@ -31,7 +31,7 @@
 
                     {{ govuk_form_element(form.get('reference_number'), {
                         'label': 'LPA reference number'| trans,
-                        'hint': 'LPA reference numbers are 12 numbers long<br/>For example, 7000-0000-0000'| trans,
+                        'hint': 'LPA reference numbers are 12 numbers long <br/>For example, 7000-0000-0000'| trans,
```
Comment (cooperaj) [inline service-front/app/src/Actor/templates/actor/lpa-add.html.twig:34]: This change doesn't seem strictly necessary and would have resulted in changes across all the translation files. Do we need it?

File: service-front/app/src/Actor/templates/actor/request-activation-key.html.twig (hunk 2 above)
Comment (cooperaj) [inline service-front/app/src/Actor/templates/actor/request-activation-key.html.twig:31]: Oh I see it matches here. If it's not too much trouble (broken translation files etc) then dropping the space would be good.

**Review by GemTay — APPROVED** [review]

**Pull request merged**